### ISSUE TITLE
Saved profiles disappears after closing the app

### ISSUE THREAD

**Issue by vishnusudheendran** [willsyo/GPU_Bot#2]
Saved profiles disappears after closing the app.

**Comment by nubonics** [comment]
it does not...

**Comment by vishnusudheendran** [comment]
Yes it does. Every time I close the app, saved profiles disappear expect the default Alaska one. I have ran birdbot and saved profiles without any issues.
